source/partials/next_release/_release.md.erb

```diff
@@ -1,35 +1,72 @@
-<h4>Improvements</h4>
+<h4>New dashboard</h4>
+
+  We have made several improvements to the new dashboard the pipeline personalization features.
```

Comment:
How about `We have made several improvements to the pipeline personalization on the new dashboard`?